### ISSUE TITLE
Release v0.3.0 does not have v0.3.0 in the package.mo

### ISSUE THREAD

**Issue by adrpo** [xrg-simulation/SMArtIInt#9]
Our OpenModelica package manager library index update job found this issue as it didn't find the v0.3.0 version.

https://github.com/xrg-simulation/SMArtIInt/commit/2610fd1c48134c0f6ef18bb20ba66cccb7af3f3c#diff-d13159208817f27a3a90cb898c40bee5835f6d030450f5d493b2604f32200f04R12

**Comment by adrpo** [comment]
```
    version="%version%",
    versionBuild = "%build%",
    versionDate = "%date%",

```

**Comment by adrpo** [comment]
I guess you forgot to run: https://github.com/xrg-simulation/SMArtIInt/blob/main/deploy.sh script before doing the release.

**Comment by casella** [comment]
@HankeXRG, please take care of this ASAP, as it is currently [breaking our package indexing system](https://test.openmodelica.org/jenkins/job/Update%20Package%20Index/).

Thanks!

**Comment by adrpo** [comment]
This is no longer critical for us. We solved the issues on our end by ignoring the `v0.3.0` tag because there is no version supporting that in `package.mo`.

You can fix this by making a new release v0.3.1 (or, not recommended, redoing the v0.3.0 tag) which has in its `package.mo` `version=0.3.1`.

**Comment by HankeXRG** [comment]
Thank you for reporting this issue. The problem is fixed in the updated release version [ v0.3.1](https://github.com/xrg-simulation/SMArtIInt/releases/tag/v0.3.1).

**Comment by casella** [comment]
Great, thanks!
